### PR TITLE
Adding a video page to the guild

### PR DESCRIPTION
**Pull request by thnukid**
image

### CODE REVIEW

**Comment by pascalw** [comment]
Cool! 👍

**Comment by thnukid** [comment]
* uses the playlist [talks](https://www.youtube.com/watch?v=u_F0O_ABPiU&list=PLlkewFOEKE8Bijbr6NHo8afBD3VIixaep), [Lunch&Learn](https://www.youtube.com/watch?v=v9wroSGYhdM&list=PLlkewFOEKE8CcZrRWzKyAE7g6XEtNdHKe) on the youtube channel

image

**Comment by pascalw** [comment]
Looks good @thnukid :+1:

I noticed there's a video in the steam by (https://www.youtube.com/watch?v=lqBMp4o--QY) which is not by a Kabisan. That probably shouldn't be in there?

File: source/stylesheets/modules/_nav.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+nav {
+  ul {
+    li {
+      display: inline;
+      float: right;
```
Comment (pascalw) [inline source/stylesheets/modules/_nav.scss]: How about:

```css
nav {
  display: flex;
  flex-direction: column;
}

ul {
  align-self: flex-end;
   display: flex;
}
```

Then you can put the `li` tags in the regular order in the source too.

Comment (thnukid) [reply source/stylesheets/modules/_nav.scss]: I have nested the nav with the ul and li element, like that 0ad6d4c - without the scope to the `nav` all `ul` items would receive the suggestion.

**Comment by thnukid** [comment]
> I noticed there's a video in the steam by (https://www.youtube.com/watch?v=lqBMp4o--QY) which is not by a Kabisan. That probably shouldn't be in there?

I have removed the talk for now.

> Looks good @thnukid 👍
Thanks for reviewing, @pascalw  💯 

image

**Comment by thnukid** [comment]
# Blog page

## Desktop

image

## Mobile
image

# Video page

## Desktop

image

## Mobile
image

**Comment by thnukid** [comment]
Can we :shipit: ? @pascalw @RoelN

**Comment by pascalw** [comment]
Looks good @thnukid! Nice work! 👍

**Comment by RoelN** [comment]
Nice job, @thnukid! 🙏

**Pull request merged**